Framework/DataHandling/test/LoadEventNexusTest.h

```diff
@@ -994,6 +994,61 @@ class LoadEventNexusTest : public CxxTest::TestSuite {
     TS_ASSERT_THROWS(loader.execute(), const InvalidLogPeriods &);
   }
 
+  void test_load_ILL_no_triggers() {
+    // ILL runs don't have any pulses, so in event mode, they are replaced in the event nexus by trigger signals.
+    // But some of these nexuses don't have any triggers either, so they are modified to be allowed to be loaded.
+
+    LoadEventNexus loader;
+
+    loader.initialize();
+    loader.setPropertyValue("Filename", "ILL/D22B/000242_trunc.event.nxs");
+    loader.setPropertyValue("OutputWorkspace", "dummy");
+    loader.setProperty("LoadAllLogs", true);
+    TS_ASSERT_THROWS_NOTHING(loader.execute());
+
+    EventWorkspace_sptr eventWS;
+    TS_ASSERT_THROWS_NOTHING(eventWS = AnalysisDataService::Instance().retrieveWS<EventWorkspace>("dummy"));
+    // Valid WS and it is an EventWorkspace
+    TS_ASSERT(eventWS);
+    if (!eventWS)
+      return;
+    TS_ASSERT_EQUALS(eventWS->getNumberEvents(), 1000);
+    TS_ASSERT_EQUALS(eventWS->run().startTime(), DateAndTime("2021-01-28T18:07:12"));
+    TS_ASSERT_EQUALS(eventWS->getPulseTimeMin(), eventWS->getPulseTimeMax());
+    TS_ASSERT_EQUALS(eventWS->getPulseTimeMin().totalNanoseconds(), 980705232000000000);
+    TS_ASSERT_DELTA(eventWS->getTofMax(), 13515.0517592763, 1e-2);
+
+    // check that the logs have been loaded by looking at some random example
+    TS_ASSERT_DELTA(eventWS->run().getPropertyAsSingleValue("reactor_power"), 43.21, 1e-2);
```

Comment:
There is no `turnDown` method in the test suite. You should cleanup the `dummy` workspaces after assertions.